srcpkgs/kwallet-cli/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'kwallet-cli'
+pkgname=kwallet-cli
+version=3.02
+revision=1
+archs="i686* x86_64*"
+wrksrc=kwalletcli
+build_style=gnu-makefile
+make_build_args="KDE_VER=5"
+hostmakedepends="pkg-config"
+makedepends="kwallet-devel qt5-devel"
+depends="kwallet qt5 mksh"
```

Comment:
kwallet and qt5 is detected automatically:

=> kwallet-cli-3.02_1: running pre-pkg hook: 04-generate-runtime-deps ...
   SONAME: libKF5Wallet.so.5 <-> kwallet>=5.26.0_1
   SONAME: libQt5Widgets.so.5 <-> qt5-widgets>=5.15.0_1